purchase_request/__manifest__.py

```diff
@@ -24,7 +24,7 @@
         "views/product_template.xml",
         "views/purchase_order_view.xml",
         "views/stock_move_views.xml",
-        "views/stock_picking_views.xml",
+        # "views/stock_picking_views.xml",
```

Comment:
remove this line and views/stock_picking_views.xml if you don't need them anymore